spec/icon_spec.lua

```diff
@@ -37,5 +37,20 @@ describe('Icon Maker', function()
 			assert.are_equal('<i class="fas fa-check" aria-hidden="true"></i>',
 				Icon.makeIcon{iconName = 'winner', screenReaderHidden = true})
 		end)
+
+		it('no additional classes', function()
+			assert.are_equal('<i class="fas fa-check"></i>',
+				Icon.makeIcon{iconName = 'winner', additionalClasses = {}})
```

Comment:
Maybe you want to define `fas fa-check` and `'winner'` as variables since they are repeated a lot?

Reply:
done